spec/ruby_llm/mcp/parameter_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+RSpec.describe RubyLLM::MCP::Parameter do
+  describe "#item_type" do
```

Comment:
Looking at your specs, we are missing the case that kicked off the issue in the first place. An array type without a item type attached. Possible to add that case?